content/7-webapp-features/1-fetch-data/angular/user.service.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { catchError, map, Subject } from 'rxjs';
+import { User } from './user';
+
+@Injectable({
+  providedIn: 'root'
+})
+
```

Comment:
The decorator stills working with this empty line ?

Reply:
Yup, completely fine.